hydromt_fiat/workflows/exposure_vector.py

```diff
@@ -1903,6 +1903,10 @@ def set_height_relative_to_reference(
             "Object ID", drop=False
         )
 
+        # Ensure that the raise_by variable has the correct type
+        if raise_by is not pd.Series:
+            raise_by = pd.Series(raise_by, index=exposure_to_modify.index)
+        
```

Comment:
This will check whether raise_by is literally the pd.Series class (by identity), but it is an instance of the class.

If you want to check an objects type, you should use isinstance(obj, class)

Reply:
good one!